### PR TITLE
Add test to main

### PR DESCRIPTION
**Pull request by bzwei**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -17,11 +16,29 @@ import (
 )
 
 func main() {
-	startRun(os.Stdin, &request.DefaultRequestHandler{})
+	var mqttURL string
+	flag.StringVar(&mqttURL, "mqtturl", "", "MQTTURL")
```
Comment (mkanoor) [inline main.go]: @bzwei we wont be running from the command line parameters anymore, instead we would have a config file with all the parameters from well known location like /etc/redhat_mqtt_client/catalog.conf

I have started this file which we can start with https://github.com/mkanoor/python_catalog_mqtt_client/blob/master/config/sample.conf

We would have to redo this area again so lets switch to the conf file.

Comment (bzwei) [reply main.go]: sounds good. should we have a default location of the conf file and overridable from command line args?

File: sample.conf [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[ANSIBLE_TOWER]
```
Comment (mkanoor) [inline sample.conf:1]: @bzwei From the on premise client we will always be connecting to the cloud.redhat.com using Mutual Authentication (mTLS) so even though we are just testing locally we need the the cert and key in the sample config file. As listed here
https://github.com/mkanoor/python_catalog_mqtt_client/blob/829a510ce3681243d8669dc09305f1551e629797/config/sample.conf#L10
As we move this to the on premise we need the cert and key. The location listed in the above sample will not change in a RHEL machine. That is the default location.

Comment (bzwei) [reply sample.conf:1]: Sure. This sample conf contains parameters used by the current implementation. We will add cert and key when we actually use them for connection.

**Review by mkanoor — APPROVED** [review]

**Pull request merged**